test/unit/specs/vislib/lib/validate_visdata.js

```diff
@@ -0,0 +1,114 @@
+define(function (require) {
+  var angular = require('angular');
+  var _ = require('lodash');
+  var $ = require('jquery');
+
+  angular.module('ValidateVisDataFactory', ['kibana']);
+
+  describe('Vislib ValidateVisData Test Suite', function () {
+    function generateData(length, size, type) {
+      function generateValues(size, type) {
+        return Array.apply(null, new Array(size))
```

Comment:
Took me a minute to figure out what was going on here. Consider switching to `_.times(size, ...)`.